src/cljs/harja/views/tilannekuva/tilannekuva.cljs

```diff
@@ -115,8 +115,7 @@
                (ikonit/livicon-chevron-down) (ikonit/livicon-chevron-right))]
             [:div.tk-checkbox-ryhma-checkbox {:on-click #(.stopPropagation %)}
              [checkbox/checkbox ryhmanjohtaja-tila-atom otsikko
-              {:width "230px" ;; 100% ei toimi tässä kontekstissa, joten määritetään leveydeksi paneelin leveys
-               :on-change (fn [uusi-tila]
+              {:on-change (fn [uusi-tila]
```

Comment:
Ei voida hardkoodata checkboks-elementin leveyttä näin, scrollbar voi muuttaa sitä ja muutenkin vähän huono idea.